framework/db/ActiveQuery.php

```diff
@@ -806,7 +806,7 @@ public function alias($alias)
      * @param string|array $condition the given conditions
      * @param string $aliasName
      */
-    private function replaceAliasPlaceholder($condition, $aliasName)
+    public function replaceAliasPlaceholder($condition, $aliasName)
```

Comment:
It's not needed but wouldn't hurt as @lynicidn suggested.
For some reasons I don't know, methods (removeDuplicatedModels, buildJoinWith, getQueryTableName, ...) are in private so I sticked to it for normalizeAliasInQuery. However I would have put it in protected.